app/routes.js

```diff
@@ -5,23 +5,22 @@ const join = require('path').join;
 
 exports.router = new express.Router();
 
-const routes = merge(config.apps.map(app_name => {
+const routes = {};
```

Comment:
I'm finding this version of the logic much more readable than the previous version with all the merges 👍